src/frontends/paddle/src/op/reduce_ops.hpp

```diff
@@ -34,17 +50,17 @@ NamedOutputs reduce_ops(const NodeContext& node) {
         }
     }
 
-    auto reduceNode = std::make_shared<T>(x, axesNode, keep_dim);
-    std::shared_ptr<Node> result = reduceNode;
+    auto reduce_node = std::make_shared<T>(x, axes_node, keep_dim);
+    std::shared_ptr<Node> result = reduce_node;
     if (scalar_output) {
         auto unsqueeze_scalar = default_opset::Constant::create(ov::element::i64, {}, {0});
-        result = std::make_shared<default_opset::Unsqueeze>(reduceNode, unsqueeze_scalar);
+        result = std::make_shared<default_opset::Unsqueeze>(reduce_node, unsqueeze_scalar);
```

Comment:
Is it redundant to add a 'unsqueeze' here and add a 'squeeze' again in the following code? Can we combine them?

Reply:
> Is it redundant to add a 'unsqueeze' here and add a 'squeeze' again in the following code? Can we combine them?

done